### PR TITLE
Character images

### PR DESCRIPTION
**Pull request by myun11**
Changed some icons.

### CODE REVIEW

File: assets/images/characters/sethos/talent-burst
Comment (man90es) [inline assets/images/characters/sethos/talent-burst:1]: This isn't Sethos's burst icon

File: assets/images/characters/candace/talent-passive-0
Comment (man90es) [inline assets/images/characters/candace/talent-passive-0:1]: This is passive 3, not 0. The rest of passives' images need to be renamed accordingly too

**Review by man90es — CHANGES_REQUESTED** [review]

File: assets/images/characters/candace/talent-passive-0
Comment (myun11) [reply assets/images/characters/candace/talent-passive-0:1]: https://genshin-impact.fandom.com/wiki/Candace#Talents

I thought "Utility Passive" is talent-passive-0, "1st Ascension Passive" is talent-passive-1 and "4th Ascension Passive" is talent-passive-2

File: assets/images/characters/sethos/talent-burst
Comment (myun11) [reply assets/images/characters/sethos/talent-burst:1]: Fixed.

File: assets/images/characters/candace/talent-passive-0
Comment (man90es) [reply assets/images/characters/candace/talent-passive-0:1]: I have no idea why that would be the case, but looking at images we already have, it apparently is

**Review by man90es — APPROVED** [review]
Looks good, thanks for your contribution!

**Pull request merged**